docs/en/integrations/data-ingestion/etl-tools/dbt/dbt-incremental-model.md

```diff
@@ -194,72 +193,27 @@ Adjust the above query to the period of execution. We leave result inspection to
-               JOIN imdb.roles ON imdb.roles.actor_id = imdb.actors.id
-               LEFT OUTER JOIN imdb.movies ON imdb.movies.id = imdb.roles.movie_id
-               LEFT OUTER JOIN imdb.genres ON imdb.genres.movie_id = imdb.movies.id
-               LEFT OUTER JOIN imdb.movie_directors ON imdb.movie_directors.movie_id = imdb.movies.id
-               LEFT OUTER JOIN imdb.directors ON imdb.directors.id = imdb.movie_directors.director_id
-       )
-       GROUP BY id
-    )
-     
-    select *
-    from actor_summary
-    where id > (select max(id) from imdb_dbt.actor_summary) or updated_at > (select max(updated_at) from imdb_dbt.actor_summary));
-    ```
+2. A new table, `actor_summary_new,` is created. The rows from the old table are, in turn, streamed from the old to new, with a check to make sure row ids do not exist in the temporary table. This effectively handles updates and duplicates.
+3. The results from the temporary table are streamed into the new `actor_summary` table:
+4. Finally, the new table is exchanged atomically with the old version via an `EXCHANGE TABLES` statement. The old and temporary tables are in turn dropped.
 
-2. The previous materialized table is renamed `actor_summary_old`. A new table `actor_summary` is created. The rows from the old table are, in turn, streamed from the old to new, with a check to make sure row ids do not exist in the temporary table. This effectively handles updates:
-    ```sql
-    insert into imdb_dbt.actor_summary ("id", "name", "num_movies", "avg_rank", "genres", "directors", "updated_at")
-    select "id", "name", "num_movies", "avg_rank", "genres", "directors", "updated_at"
-    from imdb_dbt.actor_summary__dbt_old
-    where (id) not in (select (id)
-                    from actor_summary__dbt_tmp);
-    ```
+This is visualized below:
 
-3. Finally, results from the temporary table are streamed into the new `actor_summary` table:
-    ```sql
-    insert into imdb_dbt.actor_summary ("id", "name", "num_movies", "avg_rank", "genres", "directors", "updated_at")
-    select "id", "name", "num_movies", "avg_rank", "genres", "directors", "updated_at"
-    from actor_summary__dbt_tmp;
-    ```
+<img src={require('./images/dbt_05.png').default} class="image" alt="incremental updates dbt" style={{width: '100%'}}/>
 
 This strategy may encounter challenges on very large models. For further details see [Limitations](./dbt-limitations).
 
+## Append Strategy (inserts-only mode)
 
+To overcome the limitations of large datasets in incremental models, the plugin uses the dbt configuration parameter `incremental_strategy`. This can be set to the value `append`. When set, updated rows are inserted directly into the target table (a.k.a `imdb_dbt.actor_summary`) and no temporary table is created.
+Note: Append only mode requires your data to be immutable or for duplicates to be acceptable. If you want an incremental table model that supports altered rows don’t use this mode!
```

Comment:
`requires your data to be immutable or for duplicates to be acceptable`

This is a bit awkward since immutable and allowing duplicate keys are not mutually exclusive